src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export function Tabs({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId);
+
+  const isIncorrectTabId = tabs.every(tab => tab.id !== selectedTabId);
```

Comment:
```suggestion
  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```
after that, you can remove all logic with undefined selectedTab and isIncorrectTabId